msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,117 @@
+>[!Note]
+>It is not a good idea to have multiple Teams apps link to the same AAD app ID. Team AAD app ID must be unique – if anyone tries to install and link multiple Teams app to the same AAD app ID, the installation will fail.
+
+#### 2. Check permissions configured in the 'API Permissions' section
+RSC permissions are only configured in the app manifest file. However, we need to check permissions in the 'API permissions' section in the app registration portal. Navigate to the 'API Permissions' section to check for any RSC and non-RSC permissions. If your app is newly registered, it may not have any additional permissions configured. Hence you can proceed further to step 3 to configure RSC permissions in the app manifest file. On the other side, if your app is an existing app, it may have additional permissions configured. In this case, ensure there are no RSC permissions configured here as we will configure them in the app manifest file as part of step 3. You can retain the non-RSC permissions like part of any normal app registration process.
+
+E.g. Graph permissions are not always about Teams data.
+For example, Your AAD app may need Mail.Read permission, but this is not an RSC scenario because it is not about team data, and team owners cannot give consent to other's mailboxes. APIs for Files, SharePoint, OneNote, Planner, and Calendar within a team do not support RSC yet.
+
+#### 3. Update your Teams app manifest to link to your AAD app ID
+Add a [webApplicationInfo](/microsoftteams/platform/resources/schema/manifest-schema#webapplicationinfo) section to the manifest that includes two properties - `id` is the AAD app ID, and `applicationPermissions` are the RSC permissions (scopes) your app requires.
+
+> **Note**: If your app needs non-RSC permissions, you can continue to use them as always, but they do not go in the manifest. 
+
+```
+{ 
+  "$schema": ".../MicrosoftTeams.schema.json", 
+   "webApplicationInfo": { 
+      "id": "cb38cf54-ac89-4a7a-9ea3-095d3d080037", 
+      "applicationPermissions": [  
+      "File.Read.Group",
+      "TeamSettings.Read.Group",
+      "ChannelMessage.Read.Group",
+      "Member.Read.Group"
+      ] 
+  }
+  ... 
+} 
+```
+<!Refer to the list of available Graph permissions for RSC in [Teams permissions](../graph/permissions-reference?context=graph%2Fapi%2Fbeta&view=graph-rest-beta.md#Teams permissions)section.>
+Refer to the list of available Graph permissions for RSC in <a href="https://docs.microsoft.com/en-us/graph/permissions-reference?context=graph%2Fapi%2Fbeta&view=graph-rest-beta">Teams permissions</a> section.
+When you make an RSC API call, Graph doesn't know on behalf of which user (if any) you are doing the work. Hence, you should be aware of the use cases such as creating a channel or removing a tab, as you are calling Application permissions rather than delegated permissions.
```

Comment:
The second sentence is confusing. This is also a strange place to put this note, since it has nothing to do with the manifest.

Reply:
Moved it to the beginning of the document as a note.